test/e2e/tests/conftest.py

```diff
@@ -84,6 +84,7 @@ def pytest_configure(config):
             "batch_deploy_timeout": int(os.environ.get("CORTEX_TEST_BATCH_DEPLOY_TIMEOUT", 30)),
             "batch_job_timeout": int(os.environ.get("CORTEX_TEST_BATCH_JOB_TIMEOUT", 200)),
             "async_deploy_timeout": int(os.environ.get("CORTEX_TEST_ASYNC_DEPLOY_TIMEOUT", 30)),
```

Comment:
I think this deploy timeout should match the realtime one to accommodate for image downloading and any potential instance spin ups.

Reply:
only the gateway needs to be ready... maybe the workload timeout should be increased then